### PR TITLE
Minor typographical corrections

### PR DESCRIPTION
**Pull request by 1point7point4**

### CODE REVIEW

File: docs/spec.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ For example:
  - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
 Note that these examples are not universal, in some cases different
-definitions of equivalense for that types might be more appropriate.
+definitions of equivalence for your types might be more appropriate.
```
Comment (rpominov) [inline docs/spec.md]: `that` is referring to the types from the examples above. I think we should keep it.

Comment (1point7point4) [reply docs/spec.md]: Good point.

Comment (1point7point4) [reply docs/spec.md]: Changed it to "those", just waiting on travis for a second

**Comment by rpominov** [comment]
Thanks!

**Pull request merged**